app/secrets.py

```diff
@@ -22,15 +22,12 @@ def validate_required_secrets(secrets):
         if required_secret not in secrets['secrets']:
             raise Exception("Missing Secret [{}]".format(required_secret))
 
-    validate_required_submission_keys(secrets)
 
-
-def validate_required_submission_keys(secrets):
+def validate_required_submission_keys(keys):
```

Comment:
I think you should be able to use this method from the library.
https://github.com/ONSdigital/sdc-cryptography/blob/master/sdc/crypto/key_store.py#L10